example/pubspec.yaml

```diff
@@ -22,6 +22,9 @@ dev_dependencies:
 
   flutter_aepassurance: ">= 1.0.0 <2.0.0"
 
+  flutter_aepconsent:
+    path: ../plugins/flutter_aepconsent
```

Comment:
Remember to refer to the actual plugin once it is published here. If i remember correctly i got errors when trying to use the local path from example app.

Reply:
Will change it back to actual plugin when merge to staging.

Reply:
Updated back to actual plugin version.